code/src/app/api/carInfo/route.ts

```diff
@@ -0,0 +1,364 @@
+/* eslint-disable import/prefer-default-export */
```

Comment:
There are many problems with model and route file. For example, 1) need to use unique constraint instead of manually checking each field each time, 2) these fields shouldn't be in the database at all, it should be checked dynamically or they will just kinda be false forever and etc
    oilOverdue: { type: Boolean, required: true },
    tireOverdue: { type: Boolean, required: true },
    maintenanceRequired: { type: Boolean, required: true },